src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationImageCells/TCEvaluationImageCellsHover/index.js

```diff
@@ -6,7 +6,8 @@ class TCEvaluationImageCellsHover extends Component {
     return (
       <div className="TCEvaluationImageCellsHover">
         <img width={200}
-             src={this.props.src} />
+             src={this.props.src} 
+	     alt="Can't Find" />
```

Comment:
Let's leave the alt text blank (empty string), since there isn't a good alternate explanation for the contents of the image -- I don't think "Can't Find" is quite correct, since for a user with a screen reader, the image is found, it's just that they can't see it.